src/pages/Profile/Profile.jsx

```diff
@@ -1,19 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-// import person from '../../images/person.png';
+import person from '../../images/person.png';
 // ^^ blank avatar for profile pic, uncomment the above line to use
 import './Profile.css';
 
 const Profile = ({ firstName, lastName, stats }) => {
+  const displayStats = stats.map(s => {
+    return (
+      <div key={s.description} className="stats">
+        <h4 className="number">{s.number}</h4>
+        <p className="description">{s.description}</p>
+      </div>
+    );
```

Comment:
Just a preference, but I usually keep any jsx in the return statement, so I would replace `displayStats` with `stats.map(...)`.